testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -11,11 +11,7 @@
 @pytest.fixture(scope="session")
 def cert_attributes() -> Dict[str, str]:
     """Certificate attributes"""
-    return dict(O="Organization Test",
-                OU="Unit Test",
-                L="Location Test",
-                ST="State Test",
-                C="Country Test")
+    return dict(O="Organization Test", OU="Unit Test", L="Location Test", ST="State Test", C="Country Test")
```

Comment:
This type of change is welcomed because it makes whole code shorter.

Reply:
it should be welcome, because it brings consistency (as all the black formatting) and that's main benefit of formatter